knotx-knot/knotx-knot-handlebars/src/main/java/com/cognifide/knotx/knot/templating/HandlebarsKnotVerticle.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Knot.x - Reactive microservice assembler - Handlebars Knot
+ *
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.knot.templating;
+
+import com.cognifide.knotx.knot.templating.impl.HandlebarsKnotProxyImpl;
+import com.cognifide.knotx.proxy.KnotProxy;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import io.vertx.core.eventbus.MessageConsumer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.serviceproxy.ProxyHelper;
+
+public class HandlebarsKnotVerticle extends AbstractVerticle {
```

Comment:
It looks that there is lot of similar classes that extends `AbstractVerticle` with boilerplate code. Is there a reason we don't want to have `AbstractKnot` with at least `stop()` and `start()` methods implemented?